src/functions.js

```diff
@@ -0,0 +1,183 @@
+import { completed } from './completed';
+
+export let tasks = []; // eslint-disable-line
+export const main = document.getElementById('main');
+export const listContainer = document.createElement('div');
+
+let index = 1;
+
+listContainer.id = 'list-container';
+listContainer.className = 'd-flex';
+
+function updateIndex(tasks) {
+  index = 1;
+  tasks.forEach((task) => {
+    task.index = index;
+    index += 1;
+  });
+}
+
+function getTasks() {
+  return JSON.parse(localStorage.getItem('taskArr'));
+}
+
+function saveTaskArr(tasks) {
+  localStorage.setItem('taskArr', JSON.stringify(tasks));
+  updateIndex(tasks);
+  localStorage.setItem('taskArr', JSON.stringify(tasks));
+}
+
+function checkIndex() {
+  if (localStorage.getItem('taskArr')) {
+    const newArr = JSON.parse(localStorage.getItem('taskArr'));
+    index = newArr.length + 1;
+    localStorage.setItem('index', index);
+  } else {
+    localStorage.setItem('index', '1');
+  }
+}
+
+function tasksChecker() {
+  if (localStorage.getItem('taskArr')) {
+    tasks = JSON.parse(localStorage.getItem('taskArr'));
+    return tasks;
+  }
+  tasks = [];
+  localStorage.setItem('taskArr', JSON.stringify(tasks))
+}
+
+function removeTask(tasks, index) {
+  const removing = tasks.filter((task) => task.index !== index);
+  const currentIndex = localStorage.getItem('index', index);
+  const newIndex = currentIndex - 1;
+  localStorage.setItem('index', newIndex);
+  return removing;
+}
+
+function editTask(index, inputField) {
+  const tasks = getTasks();
+  for (let i = 0; i < tasks.length; i += 1) {
+    if (tasks[i].index === index) {
+      tasks[i].description = inputField.value;
+      saveTaskArr(tasks);
+    }
+  }
+}
+
+function resetCompleted() {
+  const tasks = getTasks();
+  for (let i = 0; i < tasks.length; i += 1) {
+    if (tasks[i].completed === true) {
+      const container = document.getElementById(`task-${i + 1}`);
+      const input = document.getElementById(`input-${i + 1}`);
+      const box = document.getElementById(`cb-${i + 1}`);
+      container.classList.add('completed');
+      input.style.backgroundColor = 'lightgray';
+      input.setAttribute('disabled', 'disabled');
+      box.setAttribute('checked', 'checked');
+    }
+  }
+}
+
+function addTask(task, index, Trash, Check) {
```

Comment:
According to the clean code guidelines, when arguments of the function exceed three, it is better to use an object as the argument of the fuction. This will not only look better but also prevent mistakes in the ordering of the arguments in order for the function to work.